components/server/src/workspace/workspace-starter.ts

```diff
@@ -633,6 +633,23 @@ export class WorkspaceStarter {
             configuration.desktopIdeImage = choose.desktopIdeImage;
         }
 
+        log.info(
```

Comment:
Is it only for debugging? I think it should have `debug` log. There is a tool to enable debug log in prev envs.

I usually use `yarn telepresence` + `yarn watch` on server, and then attach with F5 and debug it.

Reply:
will remove these, they are debug log here https://github.com/gitpod-io/gitpod/pull/9467/commits/23308de53dc84a6ddca92dde75d58a1e3e34d9c5